projects/freerdp/project.yaml

```diff
@@ -0,0 +1,16 @@
+homepage: "https://www.freerdp.com/"
+language: c
+primary_contact: ""
+builds_per_day: 4
+auto_ccs:
+  - "estetus@gmail.com"
+fuzzing_engines:
+  - afl
+  - honggfuzzer
+  - libfuzzer
+sanitizers:
+  - address
+  - memory
+architectures:
+  - x86_64
+  - i386
```

Comment:
"i386" fuzzing is super optional. I'd try to get x86_64 fuzzing working first.

Reply:
removed `i386`